### ISSUE TITLE
SimpleWave example on documentation page has incorrect MMA code

### ISSUE THREAD

**Issue by rhaas80** [ianhinder/Kranc#142]
The SimpleWave.m example page on http://kranccode.org/documentation.html contains
```
groups = evolved_group};
```
which is incorrect and should instead read
```
groups = {{"evolved_group", {phi, pi}}};
```

The code shown on the web-page is also a bit out-of-sync with the code in the MMA script though I think it is only a matter of re-ordering lines.

**Comment by ianhinder** [comment]
Pull request welcome :)

**Comment by rhaas80** [comment]
Turned out to be an issue with Liquid's parsing of the the html code.

https://github.com/ianhinder/Kranc/pull/143

**Comment by ianhinder** [comment]
Closed in PR #143.
